pavelib/utils/test/suites/pytest_suite.py

```diff
@@ -161,11 +161,13 @@ def cmd(self):
                 xdist_remote_processes = self.processes
             for ip in self.xdist_ip_addresses.split(','):
                 # The django settings runtime command does not propagate to xdist remote workers
-                custom_exports = 'export MAILCHIMP_API_KEY={MAILCHIMP_API_KEY}; export TEST_DB_HOST={TEST_DB_HOST};' \
+                custom_exports = 'export MAILCHIMP_API_KEY={MAILCHIMP_API_KEY};' \
+                                 ' export MANDRILL_API_KEY={MANDRILL_API_KEY}; export TEST_DB_HOST={TEST_DB_HOST};' \
                                  ' export TEST_DB_USER={TEST_DB_USER}; export TEST_DB_PASSWORD={TEST_DB_PASSWORD};' \
                                  ' export TEST_DB_NAME={TEST_DB_NAME};'\
```

Comment:
Can we refactor it to a multi-line string, with each export statement on a separate line. Something like:

custom_exports = """export MAILCHIMP_API_KEY={MAILCHIMP_API_KEY}; 
                                    export MANDRILL_API_KEY={MANDRILL_API_KEY}; 
                                    export TEST_DB_HOST={TEST_DB_HOST}; 
                                    export TEST_DB_USER={TEST_DB_USER}; 
                                    export TEST_DB_PASSWORD={TEST_DB_PASSWORD}; 
                                    export TEST_DB_NAME={TEST_DB_NAME};
                                 """